grails-app/i18n/messages.properties

```diff
@@ -1442,6 +1443,7 @@ location.zoneAssigned.message=You cannot delete zone that is assigned to a bin l
 location.bgColor.label=Background color
 location.fgColor.label=Foreground color
 location.bgColor.invalid.matchingcolor=Colors cannot be the same
+location.supportedActivities.invalid.supportedActivities=You cannot set supported activity NONE in combination with other supported activities
```

Comment:
I'm waiting for suggestions, as I struggled with figuring out a reasonable validation message for that

Reply:
Manon says this is good.